### ISSUE TITLE
Utiliser plutôt fetch_assoc que fetch_array lors de requêtes sql en php

### ISSUE THREAD

**Issue by pzia** [v-l-m/vlm#134]
**Reported by paparazzia on 12 Apr 2009 11:25 UTC**
cela vite d'tre trop dpendant de l'ajout d'un champ dans une requte, car un code : 

```
    $query2 = "SELECT `time`,races,boats,duration,update_comment FROM updates ORDER BY `time` DESC LIMIT 1";
    $result2 = wrapper_mysql_db_query(DBNAME,$query2) or die("Query [failed \n");
    $row2 = mysql_fetch_array($result2, MYSQL_NUM);
    $lastupdate = $row2[0]($query2]);
```

devient :

```
    $query2 = "SELECT `time`,races,boats,duration,update_comment FROM updates ORDER BY `time` DESC LIMIT 1";
    $result2 = wrapper_mysql_db_query(DBNAME,$query2) or die("Query [failed \n");
    $row2 = mysql_fetch_assoc($result2, MYSQL_NUM);
    $lastupdate = $row2['time']($query2]);
```

ce qui est plus lisible et moins sensible  un changement

**Comment by pzia** [comment]
**Commented by spf on 29 Sep 2009 07:49 UTC**
(In [clarify the type of array returner in mysql_fetch_array [refs #134](1085]))

**Comment by pzia** [comment]
**Commented by spf on 29 Sep 2009 12:20 UTC**
(In [use named indexes in array (relative to [refs #134](1088])), indirectly)

**Comment by pzia** [comment]
**Commented by spf on 1 Oct 2009 09:42 UTC**
(In [move to named array indexes (better for maintenance) (partially [refs #134](1101])))

**Comment by pzia** [comment]
**Commented by spf on 1 Oct 2009 12:06 UTC**
(In [move to named array indexes (better for maintenance) (partially [refs #134](1102])))

**Comment by pzia** [comment]
**Commented by spf on 1 Oct 2009 12:29 UTC**
(In [move to named array indexes (better for maintenance) (partially [refs #134](1103]))) - continuation

**Comment by pzia** [comment]
**Commented by spf on 1 Oct 2009 16:36 UTC**
(In [[refs #134](1104]))

**Comment by pzia** [comment]
**Commented by spf on 16 Oct 2009 12:24 UTC**
(In [move to named array indexes (better for maintenance) (partially [refs #134](1193]))) - continuation

**Comment by pzia** [comment]
**Commented by paparazzia on 15 Nov 2009 19:40 UTC**
(In [1301]) \* refs #245 : utilisation du type timestamp pour la table 'updates'
- refs #134 en passant

**Comment by pzia** [comment]
**Commented by paparazzia on 10 Apr 2010 10:00 UTC**
(In [1414]) \* refs #134
- indentation

**Comment by pzia** [comment]
**Commented by paparazzia on 10 Apr 2010 10:27 UTC**
(In [1415]) refs #134

**Comment by pzia** [comment]
**Commented by paparazzia on 10 Apr 2010 10:37 UTC**
(In [1416]) \* fix #134 (le reste n'est pas fixable dans l'immediat)

**Comment by pzia** [comment]
**Commented by paparazzia on 10 Apr 2010 20:53 UTC**
(In [1418]) refs #134
